integration_tests/pages/apply/area-and-funding/exclusion-zones-and-preferred-areas/FirstPreferredAreaPage.ts

```diff
@@ -0,0 +1,25 @@
+import { Cas2v2Application as Application } from '@approved-premises/api'
+import ApplyPage from '../../applyPage'
+import { nameOrPlaceholderCopy } from '../../../../../server/utils/utils'
+import paths from '../../../../../server/paths/apply'
+
+export default class FirstPreferredAreaPage extends ApplyPage {
+  constructor(private readonly application: Application) {
```

Comment:
Should we also check for the page name being selected using `pageIsActiveInNavigation`?

Reply:
good shout, yes i will. also noticed that I've missed the other area preferences page so i will add that in too.